Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This is a development Dockerfile. For versioned Dockerfiles see:
+# https://github.com/praekeltfoundation/docker-seed
+FROM praekeltfoundation/django-bootstrap:py2.7-jessie
+
+COPY . /app
+RUN pip install -e git+git://github.com/onaio/onapie.git#egg=onapie &&\
```

Comment:
```suggestion
RUN pip install git+git://github.com/onaio/onapie.git#egg=onapie &&\
```

Reply:
If you use git to install packages then you also need to install git (`RUN apt-get-install.sh git`)